source/agora/script/Engine.d

```diff
@@ -1585,7 +1585,7 @@ unittest
     Transaction tx;
     const sig = sign(kp, tx);
 
-    const key_hash = hashFull(kp.V);
+    const key_hash = hashFull(kp.V[]);
```

Comment:
Doesn't sound right, `V` is fixed size

Reply:
Reverted now we hash using HashNoLength struct to wrap the bytes in the script.